samples/test_azure_event_hubs_receive.py

```diff
@@ -98,11 +97,12 @@ def test_event_hubs_client_proxy_settings(live_eventhub_config):
         live_eventhub_config['consumer_group'],
         live_eventhub_config['partition'])
 
-    #if not sys.platform.startswith('darwin'):  # Not sure why this passes for OSX:
+    # if not sys.platform.startswith('darwin'):  # Not sure why this passes for OSX:
     #    with pytest.raises(errors.AMQPConnectionError):
```

Comment:
Can we remove these two lines?
This passes on all platforms now?

Reply:
yes, they can all pass.
I removed these two lines.